lib/patches/ancestry_patch.rb

```diff
@@ -9,15 +9,15 @@ def update_descendants_with_new_ancestry
     end
   end
 
-  def parent=(parent)
-    super
-    clear_memoized_instance_variables
-  end
+  #  def parent=(parent)
```

Comment:
this temporary (and you are avoiding funny business) or did this break stuff?

Reply:
temporary, i think.

Reply:
i still think taking it out is necessary and for more than just testing. 

with the setter patched, we will never call any method in the gem at all.